src/os/ui/search/place/place.js

```diff
@@ -0,0 +1,59 @@
+goog.module('os.ui.search.place');
+goog.module.declareLegacyNamespace();
+
+const Feature = goog.require('ol.Feature');
+const OSObject = goog.require('os.object');
```

Comment:
This could use destructuring for specific import(s), but I'm also wondering how we want to handle case for module imports like this. From the [Google style guide](https://google.github.io/styleguide/jsguide.html#file-goog-require), I think this should use camel case (`osObject` or just `object` though I know that doesn't work when both os/goog object references are used).

Reply:
I'd agree about following the style guide, but we'll need to communicate that more widely (I hadn't seen it before today).